native-maven-plugin/src/testFixtures/groovy/org/graalvm/buildtools/maven/AbstractGraalVMMavenFunctionalTest.groovy

```diff
@@ -158,7 +181,7 @@ abstract class AbstractGraalVMMavenFunctionalTest extends Specification {
         var resultingSystemProperties = [
                 "common.repo.uri": System.getProperty("common.repo.uri"),
                 "seed.repo.uri": System.getProperty("seed.repo.uri"),
-                "maven.repo.local": testDirectory.resolve("local-repo").toFile().absolutePath
+                "maven.repo.local": testDirectory.resolve("local repo").toFile().absolutePath
```

Comment:
Why is this necessary?

Reply:
Should you add new functional test, and not just change the existing one?

Reply:
I added this for reproducing the issue.

Reply:
Okay, I asked because you changed the existing test, and now we are checking only the case with the space, right? My concern was that you removed existing check (when dir name contains `-`), and that you should maybe add a separate test for this case